tox.ini

```diff
@@ -73,6 +73,7 @@ deps =
 	coverage
 commands =
 	python -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
+	pip install pip==20.3.1
```

Comment:
Why is this needed again? Probably a remnant from before, right?

Reply:
Yes , I'll remove it from here and pin it in the requirements , this is the only uncovered one